E-Commerce/ecommerce.pgsql

```diff
@@ -0,0 +1,176 @@
+-- List all the products whose name contains the word "socks"
+SELECT
+    *
+FROM
+    products;
+
+SELECT
+    *
+FROM
+    products
+WHERE
+    product_name ILIKE '%socks%';
+
+-- List all the products which cost more than 100 showing product id, name, unit price, and supplier id
+SELECT
+    *
+FROM
+    products;
+
+SELECT
+    *
+FROM
+    product_availability;
+
+SELECT
+    p.id,
+    p.product_name,
+    pa.unit_price,
+    pa.supp_id
+FROM
+    products p
+    INNER JOIN product_availability pa ON (p.id = pa.prod_id)
+WHERE
+    unit_price > 100;
```

Comment:
I would suggest that you try to alias the names so rather than p.id, you would have something like p.id as product_id -- as this is how the column names we asked for in the question :) 

Otherwise, the query looks very good and I like that you had formatted this well!